src/main/java/net/aufdemrand/denizen/events/player/PlayerStepsOnScriptEvent.java

```diff
@@ -99,13 +99,20 @@ public ScriptEntryData getScriptEntryData() {
     }
 
     @Override
-    public HashMap<String, dObject> getContext() {
-        HashMap<String, dObject> context = super.getContext();
-        context.put("location", location);
-        context.put("previous_location", previous_location);
-        context.put("new_location", new_location);
-        context.put("cuboids", cuboids);
-        return context;
+    public dObject getContext(String name) {
+        if (name.equals("location")) {
+            return location;
+        }
+        else if (name.equals("previous_location")) {
+            return previous_location;
+        }
+        else if (name.equals("new_location")) {
+            return new_location;
+        }
+        else if (name.equals("cuboids")) {

```

Comment:
add "// Deprecated in favor of context.location.cuboids" to the end

`else if (name.equals("cuboids")) { // Deprecated in favor of context.location.cuboids`